release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2457,6 +2438,38 @@ spec:
 
 * The pipeline metrics API does not support the required `pipelinerun/taskrun` histogram values from {rh-openstack} 1.6 and beyond. Consequently, the *Metrics* tab in the *Pipeline* -> *Details* page is removed instead of displaying incorrect values. There is currently no workaround for this issue. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2074767[*BZ#2074767*])
 
+* In {product-title} {product-version}, pod security admission runs globally with privileged enforcement and restricted audit logging and API warnings. Versions of the `opm` binary released earlier than {product-title} {product-version}, do not work with the restricted profile setting. As a result, an error displays if you install a catalog source that was built with an `opm` binary released earlier than {product-title} {product-version}.
```

Comment:
```suggestion
* In {product-title} {product-version}, Pod Security Admission runs globally with privileged enforcement and restricted audit logging and API warnings. Versions of the `opm` binary released earlier than {product-title} {product-version}, do not defines the UserID on the image (i.e. in the Dockerfile we have not `USER 11211:11211 `) and in this will fail when qualified to the restricted-v2 SCC policy where the RunAsNonRoot will be set with true value as part of its strategy. 

In this way, be aware that index images built with an `opm` binary released earlier than {product-title} {product-version} before this issue can fail with the following error if applied on namespaces enforced as restricted and with the pod security admission label sychronization is disabled.
```

Reply:
@perdasilva @anik120 ^ WDYT?